src/librbd/migration/QCOWFormat.cc

```diff
@@ -32,6 +33,8 @@ namespace migration {
 #define dout_prefix *_dout << "librbd::migration::QCOWFormat: " \
                            << __func__ << ": "
 
+using boost::endian::big_to_native;
```

Comment:
Might be good to switch `rbd-nbd.cc` to `boost::endian` in this commit too.  Currently it defines its own `ntohll` macro based on `CEPH_LITTLE_ENDIAN`.

Reply:
@idryomov updated and repushed.